### PR TITLE
Version bump

### PR DESCRIPTION
**Pull request by blasten**
Also make the linter happy.

### CODE REVIEW

File: packages/multicast_dns/lib/src/native_protocol_client.dart [hunk 1]
```diff
@@ -42,7 +42,8 @@ class ResourceRecordCache {
     // Clear the cache for all name/type combinations to be updated.
     final Map<int, Set<String>> seenRecordTypes = <int, Set<String>>{};
     for (ResourceRecord record in records) {
-      seenRecordTypes[record.resourceRecordType] ??= {};
+      // TODO(dnfield): Update this to use set literal syntax when we're able to bump the SDK constraint.
+      seenRecordTypes[record.resourceRecordType] ??= {}; // ignore: prefer_collection_literals
```
Comment (dnfield) [inline packages/multicast_dns/lib/src/native_protocol_client.dart]: This should be `seenRecordTypes[record.resourceRecordType] ??= Set<String>(); // ignore: prefer_collection_literals`

Comment (blasten) [reply packages/multicast_dns/lib/src/native_protocol_client.dart]: Done.

File: packages/multicast_dns/pubspec.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ name: multicast_dns
 description: Dart package for mDNS queries (e.g. Bonjour, Avahi).
 author: Flutter Team <flutter-dev@googlegroups.com>
 homepage: https://github.com/flutter/packages/tree/master/packages/multicast_dns
-version: 0.1.0+1
+version: 0.1.1
```
Comment (dnfield) [inline packages/multicast_dns/pubspec.yaml]: We should update CHANGELOG.md to explain this.

Comment (blasten) [reply packages/multicast_dns/pubspec.yaml]: ah. forgot that. Done.

File: packages/multicast_dns/CHANGELOG.md [hunk 3]
```diff
@@ -2,4 +2,8 @@
 
 * Initial Open Source release.
 * Migrates the dartino-sdk's mDNS client to Dart 2.0 and Flutter's analysis rules
-* Breaks from original Dartino code, as it does not use native libraries for macOS and overhauls the `ResourceRecord` class.
\ No newline at end of file
+* Breaks from original Dartino code, as it does not use native libraries for macOS and overhauls the `ResourceRecord` class.
+
+## 0.1.1
```
Comment (dnfield) [inline packages/multicast_dns/CHANGELOG.md]: Nit: we usually put these in descending order (so this should be on top) so the user sees the newest things first.

Comment (blasten) [reply packages/multicast_dns/CHANGELOG.md]: makes sense!

**Pull request merged**